onnxruntime/core/optimizer/transformer_memcpy.cc

```diff
@@ -322,6 +351,8 @@ void TransformerMemcpyImpl::ProcessInitializers(const KernelRegistryManager& ker
 
     p_node->ReplaceDefs(dup_replacements);
   }
+
+  return !replacements.empty();
 }
```

Comment:
A comment saying this indicates the graph was modified would be helpful for someone new to the code